src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/Checker.java

```diff
@@ -701,44 +706,6 @@ public Void visitAttribute(AttributeTree tree, Void ignore) {
         return null;
     }
 
-    private void validateHtml4Attrs(AttributeTree tree, Name name, AttrKind k) {
-        switch (k) {
-            case ALL:
-            case HTML4:
-                break;
-
-            case INVALID:
-                env.messages.error(HTML, tree, "dc.attr.unknown", name);
-                break;
-
-            case OBSOLETE:
-                env.messages.warning(HTML, tree, "dc.attr.obsolete", name);
-                break;
-
-            case USE_CSS:
-                env.messages.warning(HTML, tree, "dc.attr.obsolete.use.css", name);
-                break;
-
-            case HTML5:
-                env.messages.error(HTML, tree, "dc.attr.not.supported.html4", name);
-                break;
-        }
-    }
-
-    private void validateHtml5Attrs(AttributeTree tree, Name name, AttrKind k) {
-        switch (k) {
-            case ALL:
-            case HTML5:
-                break;
-
-            case INVALID:
-            case OBSOLETE:
-            case USE_CSS:
-            case HTML4:
-                env.messages.error(HTML, tree, "dc.attr.not.supported.html5", name);
-                break;
-        }
-    }
 
     private boolean checkAnchor(String name) {
```

Comment:
I was going to let it slide for this round of cleanup, but if you're editing this file again (see comment on line 736) it might be worth changing the use of `anchor` to `id`.  `anchor` is a term that was more appropriate in the days before the `id` attribute, when we used `<a name="...">`.  This is an optional suggestion.  It might equally be worth focussing on the must-fix items, and postpone this cleanup for later.

Reply:
I understand.  But is it really no problem to be done in part of the cleanup of doclint?  
Looking at the classes in jdk/javadoc/internal/doclint,  the term `(anchor|Anchor)` looks like only used in Checker.java and resource files.  But a lot of other files, for instance in jdk/javadoc/internal/doclets, use this term to refer to the `id` or `name` attribute.  I would be fine if it is supposed to be done in each cleanup in the future.

Reply:
If we would apply similar changing to doclint.properties, what you are thinking is like below right?

```
-dc.anchor.already.defined = anchor already defined: "{0}"
-dc.anchor.value.missing = no value given for anchor
+dc.id.already.defined = attribute "id" already defined: "{0}"
+dc.id.value.missing = no value given for attribute "id"

-dc.invalid.anchor = invalid name for anchor: "{0}"
-dc.invalid.id = invalid name for attribute "id": "{0}"
```